bin/varnishd/mgt/mgt_param.c

```diff
@@ -968,3 +968,84 @@ MCF_DumpRstParam(void)
 		printf("\n\n");
 	}
 }
+
+void
+MCF_DumpJsonParam(void)
+{
+	struct plist *pl;
+	const struct parspec *pp;
+	const char *p, *q;
+	unsigned flags;
+	struct vsb *vsb = VSB_new_auto();
+
+	CHECK_OBJ_NOTNULL(vsb, VSB_MAGIC);
+
+	p = "\n";
+	VSB_cat(vsb, "{");
+	VSB_indent(vsb, 2);
```

Comment:
You should use the VBOB API to build your object model and the VBOR API to turn it into JSON.